drgn/helpers/linux/bpf.py

```diff
@@ -180,3 +183,59 @@ def cgroup_bpf_prog_for_each_effective(
             if not prog:
                 break
             yield prog
+
+
+def bpf_prog_used_maps(bpf_prog: Object) -> Iterator[Object]:
+    """Yield maps used by a BPF program."""
```

Comment:
The convention for helpers is to document the C type of object arguments/return values:
```suggestion
    """
    Yield maps used by a BPF program.

    :param bpf_prog: ``struct bpf_prog *``
    :return: Iterator of ``struct bpf_map *`` objects.
    """
```